dht_bootstrap.go

```diff
@@ -17,7 +19,7 @@ var DefaultBootstrapPeers []multiaddr.Multiaddr
 
 // Minimum number of peers in the routing table. If we drop below this and we
 // see a new peer, we trigger a bootstrap round.
-var minRTRefreshThreshold = 4
+var minRTRefreshThreshold = 20
```

Comment:
this seems like a high minimum for triggering new bootstraps. was this for testing, or intentional?

Reply:
This is intentional. The current interval for refresh is too much (1 hour) & we want to be more proactive in doing refresh when needed rather than doing them to often without needing to do them. But really, we can always tune these params if we run into problems later.